packages/tools/tests/dummy/app/templates/components/cardstack/post-isolated.hbs

```diff
@@ -18,8 +18,13 @@
   <ul>
     {{#each content.comments as |comment|}}
       <li>
-        {{cs-field comment.poster 'name'}}: {{cs-field comment 'body'}}
-        {{#cs-field-group comment name="karma" caption="Karma" fields="karmaValue,karmaType" as |ck|}}
+        {{cs-field comment.poster "name"}}: {{cs-field comment "body"}}
+        {{#cs-field-group
+          comment
+          name="karma"
+          caption="Karma"
+          fields="karmaValue,karmaType"
+        as |ck|}}
```

Comment:
seems unrelated?

Reply:
It is. This was one of the merge conflicts I had to resolve manually during the interactive rebase and I decided to resolve with the version that also passes the template lint (probably that's also fixed in the template lint PR I'm submitting soon). This is just a "refactor", nothing was changed.